packages/app/src/cli/models/organization.ts

```diff
@@ -18,6 +18,12 @@ export interface MinimalAppIdentifiers {
   organizationId: string
 }
 
+export interface MinimalAppIdentifiersPossiblyExcludingId {
```

Comment:
But do we need the ID here at all? can't we just remove it from the type?

Reply:
If we have ID available, we would prefer to use it. However, we need to be flexible enough to handle a case when ID is unavailable but we have API key.

If you look at the logic [here](https://github.com/Shopify/cli/pull/4918/files#diff-06b22225f82056c865dd6d82b1c00f8ce938cea2285962890542d5581bd6d429R802), this will make more sense.

It's possible we could just always use API key as identifier, but we are using ID in all the other mutations (e.g. create version), so consistency seems preferable.